### DISCUSSION TITLE
Is it possible to get this on Bukkit 1.5.2?

### DISCUSSION THREAD

**Discussion by Hellscaped** [SkriptLang/Skript#5031] [General]
I'm running an Eaglercraft server, its a web port of java 1.5.2. It has a modded version of bukkit 1.5.2 that lets you join from an EC client. I'm just looking for a way to get this on there, as this was useful on my normal servers.

**Comment by bluelhf** [comment]
You could run a version of Skript from when 1.5.2 was released — don't expect much support for any bugs, though...

You can find a 1.5.2 release from 2013 [here](https://dev.bukkit.org/projects/skript/files/742345).

**Comment by TPGamesNL** [comment]
We only support 1.9+. As said above, feel free to use older versions, but you won't get support for them.
